webpack.config.js

```diff
@@ -31,20 +29,31 @@ module.exports = {
         })
     ],
     module: {
-        loaders: [
+        preLoaders: [
             {
-                test: /\.jsx?$/,
+                test: /\.js$/,
                 exclude: /node_modules/,
-                loader: 'babel'
-            },
-            {
-                test: /\.json?$/,
-                loader: 'json'
-            },
-            {
-                test: /\.scss$/,
-                loader: 'style!css!sass?modules&localIdentName=[name]---[local]---[hash:base64:5]'
+                loader: 'eslint'
             }
-        ]
+        ],
+        loaders: [
+        {
+            test: /\.js$/,
+            exclude: /node_modules/,
+            loader: 'babel'
+        }, {
+            test: /\.json?$/,
+            loader: 'json'
+        }, {
+            test: /\.scss$/,
+            loader: 'style!css!sass?modules&localIdentName=[name]---[local]---[hash:base64:5]'
```

Comment:
i suggest you re-write one-line syntax for loader to array with included loaders